TTHAnalysis/python/tools/higgsDiffRecoTTH.py

```diff
@@ -152,7 +157,7 @@ def analyze(self, event):
                     Hvisconstr = lep+Wconstr
                     mHvisconstr = Hvisconstr.M()
                     pTHvisconstr = Hvisconstr.Pt()
-                    if mHvisconstr<self.cuts_mH_vis[0] or mHvisconstr>self.cuts_mH_vis[1]: continue
+#                    if mHvisconstr<self.cuts_mH_vis[0] or mHvisconstr>self.cuts_mH_vis[1]: continue
```

Comment:
Is this the latest decision we made? If it's just commented because of a test, please uncomment it again, otherwise it's fine as it is.